Source/Session/Session.swift

```diff
@@ -10,7 +10,9 @@ public class Session: NSObject {
     
     public let webView: WKWebView
     public var pathConfiguration: PathConfiguration?
-    
+
+    var isSnapshotCacheStale = false
```

Comment:
Could this be made public or could a method be exposed on Session to mark is as needing clearing? I'm not _that_ familiar with Swift visibility, but I assume that this var is not available outside of the package?

We're building on top of Turbo iOS in the React Native adapter and it'd be great to be able to mark session snapshots as stale.

Reply:
Thinking about, I believe it would make sense to expose a method - what if you'd like to mark a Session snapshot cache as stale from a native controller in an app with both web screens and native screens? Would that be possible in the current implementation?

Reply:
Correct, if `public` or `private` isn't declared then `internal` is assumed which makes the property only accessible to turbo-ios.

Exposing this publicly via a function sounds like the right approach to me. Especially because it could align well with #174 and #150.

Let me think on the best approach to incorporate all three.

Reply:
> Exposing this publicly via a function sounds like the right approach to me. Especially because it could align well with #174 and #150.

Yeah, agree! :)

This PR is pointing to the `turbo-navigator` branch - would it make sense to split this PR up so that the changes to `Session` would be a separate PR that could be merged to `main`? I think we also need a corresponding PR in `turbo-android` to [keep feature parity](https://github.com/hotwired/turbo-ios/commit/bad1d5582951a58c2a498f9f30260d1b7a00085c), right?